crypto/batchverifier.go

```diff
@@ -63,31 +62,23 @@ func ed25519_randombytes_unsafe(p unsafe.Pointer, len C.size_t) {
 	RandBytes(randBuf)
 }
 
-// MakeBatchVerifierWithAlgorithmDefaultSize create a BatchVerifier instance. This function pre-allocates
-// amount of free space to enqueue signatures without expanding. this function always use the batch
-// verification algorithm
-func MakeBatchVerifierWithAlgorithmDefaultSize() *BatchVerifier {
-	return MakeBatchVerifier(minBatchVerifierAlloc, true)
-}
-
 // MakeBatchVerifierDefaultSize create a BatchVerifier instance. This function pre-allocates
 // amount of free space to enqueue signatures without expanding
-func MakeBatchVerifierDefaultSize(enableBatchVerification bool) *BatchVerifier {
-	return MakeBatchVerifier(minBatchVerifierAlloc, enableBatchVerification)
+func MakeBatchVerifierDefaultSize() *BatchVerifier {
+	return MakeBatchVerifier(minBatchVerifierAlloc)
 }
 
 // MakeBatchVerifier create a BatchVerifier instance. This function pre-allocates
 // a given space so it will not expaned the storage
-func MakeBatchVerifier(hint int, enableBatchVerification bool) *BatchVerifier {
+func MakeBatchVerifier(hint int) *BatchVerifier {
```

Comment:
I would go even farther on this rename. I would name the one that takes no arguments `MakeBatchVerifier`, and the other `MakeBatchVerifierWithHint` since the one with no arguments is the one used almost everywhere, it ought to have the nice short name.